src/styles/globals.css

```diff
@@ -1,4 +1,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Work+Sans&display=swap');
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+html {
+    font-size: calc(15px + 0.390625vw);
+}
```

Comment:
It looks like this caused everything to zoom in a lot when I tried it locally. Was this intended?

Before

![Screenshot from 2023-03-14 22-20-47](https://user-images.githubusercontent.com/46171819/225214725-022ed579-dd92-4f83-bea6-80ae60bc61c7.png)

After

![Screenshot from 2023-03-14 22-20-59](https://user-images.githubusercontent.com/46171819/225214754-c3cb0169-04cc-476e-bf35-527856f6b1c3.png)

Reply:
Yeah, we are using 'rem' units which scale with document's font-size. And to really enable the responsive design, we have to set the font-size depending on the viewport width. I found this calculation scales pretty well for all screens